### PR TITLE
Pool card UI cleanup

### PR DESCRIPTION
**Pull request by lazarmitic**

### CODE REVIEW

File: packages/tempus-client_v2/src/components/shared/style.scss [hunk 1]
```diff
@@ -7,7 +7,6 @@
   border: 1px solid #f1f1f1; // TODO add to constants
   box-shadow: 2px 2px 4px rgba(0, 0, 0, 0.25); // TODO add to constants
   border-radius: 5px;
-  height: 195px;
```
Comment (danieledamiani) [inline packages/tempus-client_v2/src/components/shared/style.scss:10]: has this been tested with the other panels? (Terms and Current Position)?

File: packages/tempus-client_v2/src/components/shared/style.scss [hunk 2]
```diff
@@ -22,7 +21,7 @@
   justify-content: space-between;
   align-items: center;
   width: 100%;
-  padding: 10px 0 4px;
```
Comment (danieledamiani) [inline packages/tempus-client_v2/src/components/shared/style.scss:25]: same as above

File: packages/tempus-client_v2/src/components/pool/Pool.tsx [hunk 3]
```diff
@@ -2,29 +2,51 @@ import { FC } from 'react';
...
   return (
     <div className="tc__pool">
-      <Typography variant="h1">{getText('pool', language)}</Typography>
+      <Typography variant="card-title">{getText('pool', language)}</Typography>
       <div className="tc__pool__body">
         <div className="tc__pool__body__item">
-          <Typography variant="h5">{getText('marketImpliedYield', language)}</Typography>
-          <Typography variant="h5">5.52%</Typography>
+          <div className="tc__pool__body__item-title">
+            <Typography variant="card-body-text" color="title">
```
Comment (danieledamiani) [inline packages/tempus-client_v2/src/components/pool/Pool.tsx:18]: `H1` wasn't good?

**Review by danieledamiani — CHANGES_REQUESTED** [review]
questions more than comments

File: packages/tempus-client_v2/src/components/shared/style.scss (hunk 1 above)
Comment (lazarmitic) [reply packages/tempus-client_v2/src/components/shared/style.scss:10]: Checked, they look good with all of the changes.

File: packages/tempus-client_v2/src/components/pool/Pool.tsx (hunk 3 above)
Comment (lazarmitic) [reply packages/tempus-client_v2/src/components/pool/Pool.tsx:18]: It didn't match the design sizes.

**Review by danieledamiani — APPROVED** [review]

**Pull request merged**